tools/External Tools.xml

```diff
@@ -72,8 +72,8 @@
   <tool name="PCLint" showInMainMenu="false" showInEditor="false" showInProject="false" showInSearchPopup="false" disabled="false" useConsole="true" showConsoleOnStdOut="false" showConsoleOnStdErr="true" synchronizeAfterRun="true">
     <exec>
       <option name="COMMAND" value="ament_pclint" />
-      <option name="PARAMETERS" value="$FilePath$" />
-      <option name="WORKING_DIRECTORY" value="$FileDir$" />
+      <option name="PARAMETERS" value="--check-headers --package-root $FileParentDir$ $FileParentDir$" />
```

Comment:
@left4taco There's two issues here:

1. If you are trying to analyze a single file in a package, it will actually analyze all the files in the package
2. If the `src` folder has any nested directories (e.g. see `rclcpp`), then the package root will be wrong

It looks like there isn't a variable to get the path of CMakeLists.txt, so I think the best alternative is:
`--check-headers --package-root $CMakeCurrentBuildDir$/.. $FilePath$`

Reply:
@jpsamper2009 I changed it to `$FilePath$`. Developers have to click the package in left navi panel to check the whole package.